other/deny-creation-of-resources/deny-creation-of-resources.yaml

```diff
@@ -0,0 +1,44 @@
+apiVersion: kyverno.io/v1
+kind: ClusterPolicy
+metadata:
+  name: deny-creation-of-resources
```

Comment:
Suggest a different name. This isn't just about denying creation but also updates as well.